test-cases/repair/bootstrap-false.yaml

```diff
@@ -0,0 +1,13 @@
+test_duration: 1440
```

Comment:
Move the file to "test-cases/performance/" change the name to "perf-regression-bootstrap"